src/m365/aad/commands/o365group/o365group-add.ts

```diff
@@ -179,6 +138,95 @@ class AadO365GroupAddCommand extends GraphCommand {
       }, (rawRes: any): void => this.handleRejectedODataJsonPromise(rawRes, logger, cb));
   }
 
+
+  private validateOwners(logger: Logger, args: CommandArgs): Promise<any> {
+    if (!args.options.owners) {
+      if (this.debug) {
+        logger.logToStderr('Owners not set. Skipping');
+      }
+      return Promise.resolve(undefined as any);
```

Comment:
Let's not use any and instead return an empty array to avoid issues with code upstream.